NodeJS/src/AddressBookClient.ts

```diff
@@ -0,0 +1,46 @@
+import { SkillPriority } from './models'
+import { RestClient } from './RestClient'
+import { sleep } from './Utils'
+import { AddressBookResponse } from './models/AddressBookResponse'
+import { AddressBookSearchRequest } from './models/AddressBookSearchRequest'
+import { AddressBookSearchResponse } from './models/AddressBookSearchResponse'
+
+export class AddressBookClient {
+  private restClient: RestClient
+  private subAccountId: string
+
+  constructor(subAccountId: string, restClient: RestClient) {
+    this.restClient = restClient
+    this.subAccountId = subAccountId
```

Comment:
@yangAtSpoken, correct me if I'm wrong, but isn't this supposed to be
```suggestion
    this.subAccountAppId = subAccountAppId
```

Reply:
my bad. I was AgentClient as an example which needs clientId for its requests.
for AddressBookClient I don't need clientId, so I'll remove this.

Reply:
done